src/styl/index.styl

```
@@ -10,6 +10,12 @@ section.main
 	flex-direction column
 	margin 0 auto
 
+	.controls
+		flex-wrap wrap
+
+		@media screen and (max-width: 440px)
+			justify-content right
```

```suggestion
 			justify-content left
```

What was the reason for specifying right here?

---

My bad, I though you said right-align 😅
Though having the controls aligned to the right does feel a bit more natural (to me at least).
Do you still want me to change this?

---

My bad 😅